src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -49,13 +50,28 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
 
             // If we need the host (if it exists), use it, otherwise target the console app directly.
-            string testConsoleAppArgs = "\"" + a.FullName + "\" " + t.FullName + " " + method.Name + " " + string.Join(" ", args);
+            var testConsoleAppArgs = new StringBuilder();
+            testConsoleAppArgs.Append(ExtraParameter);
```

Comment:
>I added an option earlier that controls whether we quote each argument (on by default)

Rather than making it an option, I would just add a separate overload (**RemoteInvokeRaw(method, String rawCommandLine, options)**) for the tiny subset of tests that actually care about passing untokenized command lines. I think that would be the Environment.CommandLine tests and System.Diagnostic.Process tests that focus on testing tokenization and nothing else.

For everyone else, it should "just work" - they pass in an array of arguments containing arbitrary characters and RemoteInvoke owns tokenizing it in a roundtrippable way.